wiki/Article_Style_Guide/en.md

```diff
@@ -370,104 +416,127 @@ Please limit to a level of four (4) sub-lists at a time (if you need more, you s
 
 ## Code
 
-Use code (inline) when noting:
-
-- a key on the keyboard
-  - should use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
-  - examples:
-    - `.`
-    - `Alt`
-    - `Ctrl` + `Shift` + `A`
-- the text of a button
-  - examples:
-    - `1. Let's do it.`
-    - `4. Noooo! I didn't mean to!`
-- the name of a folder
-  - examples:
-    - `Exports`
-    - `Downloads`
-- the name of a directory
-  - examples:
-    - `/Songs/`
-    - `/Skins/User/`
-- the name of a keyword
-  - examples:
-    - `ComboBurstStyle`
-    - `[Events]`
-- the name of a file extension
-  - must add a period (`.`) then the file extension in lowercase letters.
-  - examples:
-    - `.jpg`
-    - `.gif`
-    - `.osz`
-- the name of a chat channel
-  - examples:
-    - `#lobby`
-    - `#osu`
+Any key on the keyboard should use inline code.
+When doing so, it should use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+For example:
+
+- `` `.` ``
+- `` `Alt` ``
+- `` `Ctrl` + `Shift` + `A` ``
+
+When referring to the text of a button or menu, use inline code.
```

Comment:
This was initially a list but that gave me a headache so I made this long repeating mess. I will make subsections instead.